src/main/java/com/devcycle/sdk/server/local/api/DVCLocalClient.java

```diff
@@ -201,8 +201,10 @@ public void close() {
     if (!isInitialized) {
       return;
     }
-    configManager.cleanup();
-    eventQueueManager.cleanup();
+    if (configManager != null)
+      configManager.cleanup();
+    if(eventQueueManager != null)
+      eventQueueManager.cleanup();
```

Comment:
for style consistency: 

```suggestion
    if (configManager != null) {
      configManager.cleanup();
    }
    if (eventQueueManager != null) {
      eventQueueManager.cleanup();
    }
```